### PR TITLE
welcomes new team members :)

### PR DESCRIPTION
**Pull request by acityinohio**

### CODE REVIEW

File: _posts/blog/2020-06-03-welcome-jane-and-teor.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+At the Foundation, Jane will be primarily focused on
+[Zebra](https://github.com/ZcashFoundation/zebra), our consensus-compatible
+Rust-rewrite Zcash node software, while she will also bring her extensive
+experience with the Rust community to guide future open source contribution and
+development strategy.
+
+## Meet Teor
+
+Teor worked as a software engineer on privacy-preserving protocols and
```
Comment (teor2345) [inline _posts/blog/2020-06-03-welcome-jane-and-teor.md:37]: Grammar tweak. I'm ok either way, but I think the flow is slightly better with this change.

```suggestion
Teor has worked as a software engineer on privacy-preserving protocols and
```

Comment (teor2345) [reply _posts/blog/2020-06-03-welcome-jane-and-teor.md:37]: Do I need to mention languages here?
It was mainly C, but I've also worked in Python and Rust, in roughly that order.

(I don't mind either way, and I'm happy to leave them out.)

Comment (acityinohio) [reply _posts/blog/2020-06-03-welcome-jane-and-teor.md:37]: @teor2345 your preference! If you'd like to add go for it, and I'll commit later tonight/tomorrow morning.

Comment (teor2345) [reply _posts/blog/2020-06-03-welcome-jane-and-teor.md]: I think I'm good, it doesn't flow well in my head. And people can work it out if they want :-)

File: _posts/blog/2020-06-03-welcome-jane-and-teor.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+At the Foundation, Jane will be primarily focused on
+[Zebra](https://github.com/ZcashFoundation/zebra), our consensus-compatible
+Rust-rewrite Zcash node software, while she will also bring her extensive
+experience with the Rust community to guide future open source contribution and
+development strategy.
+
+## Meet Teor
+
+Teor has worked as a software engineer on privacy-preserving protocols and
```
Comment (teor2345) [inline _posts/blog/2020-06-03-welcome-jane-and-teor.md:37]: It's worth mentioning that I was a researcher as well as a developer.
```suggestion
Teor has worked as a software engineer and researcher on privacy-preserving protocols and
```

**Comment by acityinohio** [comment]
Excellent, I will merge tomorrow morning ET and promote then, thanks for the review and input to you both!

**Pull request merged**